.github/renovate.json

```diff
@@ -1,7 +1,70 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "enabledManagers": ["github-actions"],
+  "enabledManagers": ["github-actions", "npm"],
   "prConcurrentLimit": 5,
   "rebaseWhen": "conflicted",
-  "schedule": ["every weekday before 5am"]
+  "schedule": ["every weekday before 5am"],
+  "packageRules": [
+    {
+      "enabled": false,
+      "matchManagers": ["npm"],
+      "rangeStrategy": "bump",
+      "stabilityDays": 3
+    },
+    {
+      "enabled": true,
+      "matchManagers": ["npm"],
+      "matchFiles": ["package.json"]
+    },
+    {
+      "enabled": true,
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^cross-env$", "^dotenv-flow$", "^wait-on$"]
+    },
```

Comment:
These package appear in the root package.json file but also in some workspaces. If we don't include them, their versions will diverge between the workspaces and `yarn lint:dependency-version-consistency` will fail in CI.